tests/scripts/pulp_file/test_acs.sh

```diff
@@ -38,7 +38,7 @@ test "$(echo "$OUTPUT" | jq ".paths | length")" -eq 2
 
 # test refresh
 expect_succ pulp --background file acs refresh --acs $acs
-task_group=$(echo "$ERROUTPUT" | grep -E -o "${PULP_API_ROOT}api/v3/task-groups/[-[:xdigit:]]*/")
+task_group=$(echo "$ERROUTPUT" | grep -E -o "/[/a-zA-Z0-9_]*/api/v3/task-groups/[-[:xdigit:]]*/")
```

Comment:
This regex will allow some really odd URLs. 

PULP_API_ROOT starts and ends with /, domain-name is "just" the name. Can we put an if-domain switch here, with something like 

```suggestion
task_group=$(echo "$ERROUTPUT" | grep -E -o "${PULP_API_ROOT}{$domain_name}/api/v3/task-groups/[-[:xdigit:]]*/")
```

if-domain, and the existing check if-not-domain?

Reply:
It is also de facto of a subtle check that the href has the expected format. I think the grep would fail our test. I do not feel stong either way. If you simplify, however we can just use `.*`.

Reply:
```suggestion
task_group=$(echo "$ERROUTPUT" | grep -E -o "${PULP_API_ROOT}\(/[_a-zA-Z0-9]*\)?/api/v3/task-groups/[-[:xdigit:]]*/")
```